simplq/src/components/pages/Home/MyQueues.jsx

```diff
@@ -2,20 +2,32 @@ import React from 'react';
 import DeleteIcon from '@material-ui/icons/Delete';
 import IconButton from '@material-ui/core/IconButton';
 import { useHistory } from 'react-router';
+import { useSelector, useDispatch } from 'react-redux';
 import styles from '../../../styles/homePage.module.scss';
-import { useSelector } from 'react-redux';
+import { deleteQueue, getMyQueues } from '../../../services/queue';
+import { handleApiErrors } from '../../ErrorHandler';
+import { setMyQueues } from '../../../store/appSlice';
 
 export default () => {
   const history = useHistory();
+  const dispatch = useDispatch();
   const myQueues = useSelector((state) => state.appReducer.myQueues);
 
-  if (myQueues.length === 0) {
-    return null;
-  }
+  const onDeleteClick = (queueId) => {
+    deleteQueue(queueId)
+      .then(() => getMyQueues().then((myNewQueues) => dispatch(setMyQueues(myNewQueues))))
+      .catch((err) => handleApiErrors(err))
+      .then(() => history.push('/'))
+      .catch((err) => handleApiErrors(err));
+  };
 
```

Comment:
Moving this to utilFns would be a good idea.

Reply:
In a PR, we can remove all `.catch((err) => handleApiErrors(err))` globally, and use a axios interceptor to handle this in one place.

Reply:
I actually meant moving the onDeleteClick fn since it's used in MyQueues and DeleteQueues

Reply:
As for the interceptor, I'll look at the error boundary and interceptor probably this weekend.

Reply:
Done. I have moved this into the queue service.